index.html

```diff
@@ -1,11 +1,167 @@
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Homework 2</title>
 </head>
-<body>
-  
+
+<body class="page__body">
+  <header>
+    <a class="logo" href="index.html">
+      <span class="logo__web">Web</span>
+      <span class="logo__studio logo__studio--black">Studio</span>
+    </a>
+    <ul class="navigation">
+      <li>
+        <a class="navigation__link" href="#">Студія</a>
+      </li>
+      <li>
+        <a class="navigation__link" href="portfolio.html">Портфоліо</a>
+      </li>
+      <li>
+        <a class="navigation__link" href="#">Контакти</a>
+      </li>
+    </ul>
+    <div class="contacts">
+      <a class="contacts__link" href="mailto:info@devstudio.com">
+        info@devstudio.com
+      </a>
+      <a class="contacts__link" href="tel:+380961111111">
+        +38 096 111 11 11
+      </a>
+    </div>
+  </header>
+
+  <main>
+    <section class="hero page__section">
+      <h1 class="hero__main-title">
+        Ефективні рішення для вашого бізнесу
+      </h1>
+      <button class="hero__button">Замовити послугу</button>
+    </section>
+
+    <section class="features">
+      <ul class="features__list">
+        <li class="features__item feature">
+          <h3 class="features__title">
+            Увага до деталей
+          </h3>
+          <p class="features__text">
+            Идейні міркування, і навіть початок повсякденної роботи з формування позиції.
+          </p>
+        </li>
+        <li>
+          <h3 class="features__title">
+            Пунктуальність
+          </h3>
+          <p class="features__text">
+            Завдання організації, особливо рамки і місце навчання кадрів тягне у себе.
+          </p>
+        </li>
+        <li>
+          <h3 class="features__title">
+            Пунктуальність
+          </h3>
+          <p class="features__text">
+            Так само консультація з широким активом значною мірою зумовлює.
+          </p>
+        </li>
+        <li>
+          <h3 class="features__title">
+            Сучасні технології
+          </h3>
+          <p class="features__text">
+            Значимість цих проблем настільки очевидна, що реалізація планових завдань.
+          </p>
+        </li>
+      </ul>
+    </section>
+
+    <section class="services">
+      <h2 class="services__title">
+        Чим ми займаємося
+      </h2>
+      <ul class="services__list">
+        <li>
+          <img src="images/Services/keyboard.jpg" alt="Чим мі займаємося" width="370" height="294">
+        </li>
+        <li>
+          <img src="images/Services/phone.jpg" alt="Чим мі займаємося" width="370" height="294">
+        </li>
+        <li>
+          <img src="images/Services/rpb.jpg" alt="Чим мі займаємося" width="370" height="294">
+          <!--Изображения выше имеют нужное разрешение, поэтому ни ширину, ни высоту менять не стал-->
+        </li>
+      </ul>
+    </section>
+
+    <section class="team">
+      <h2 class="team__title services__title">
```

Comment:
this is not `services` title. If you want to add similar styles, create a `.section` block and set rules for `.section__title`, for example, and apply it to every section title if you like
```suggestion
      <h2 class="team__title section__title">
```